.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,7 +1,21 @@
 *Issue #, if available:*
 
+Note: If merging this PR should also close the associated Issue, please also add that Issue # to the Linked Issues section on the right. 
+
+### Description
+
+### Tests run
+
+### DLC image/dockerfile
+
+### Additional context
+
+## Label Checklist
+- [ ] I have added the project label for this PR (*<project_name>* or "Improvement")
```

Comment:
then labels should be present for that project, some may not have access to create those.

Reply:
How do you propose we go about this, apart from creating the labels whenever a new project starts?